src/resolvers/ReviewResolver.ts

```diff
@@ -0,0 +1,277 @@
+                });
+                await dropDown.save();
+            }
+        }
+
+        return question;
+    }
+
+    @Mutation(() => ReviewQuestion)
+    @UseMiddleware(isAuth)
+    async updateQuestion(@Arg("input") input: UpdateQuestionInput) {
+        const question = await ReviewQuestion.findOne({ where: { id: input.id } });
+        if (!question) {
+            throw new Error("Question not found");
+        }
+
+        if (input.question !== undefined) question.question = input.question;
+        if (input.questionType !== undefined) question.questionType = input.questionType;
+        if (input.rangeMin !== undefined) question.rangeMin = input.rangeMin;
+        if (input.rangeMax !== undefined) question.rangeMax = input.rangeMax;
+
+        await question.save();
+
+        // Update dropdown options if provided
+        if (input.dropDown) {
+            // Remove existing dropdown options
+            await DropDown.delete({ reviewQuestion: { id: question.id } });
+
+            // Create new dropdown options
+            for (const dropDownInput of input.dropDown) {
+                const dropDown = DropDown.create({
+                    text: dropDownInput.text,
+                    value: dropDownInput.value,
+                    reviewQuestion: question
+                });
+                await dropDown.save();
+            }
+        }
+
+        return question;
+    }
+
+    @Mutation(() => Boolean)
+    @UseMiddleware(isAuth)
+    async deleteQuestion(@Arg("id") id: string) {
+        const question = await ReviewQuestion.findOne({ where: { id } });
+        if (!question) {
+            throw new Error("Question not found");
+        }
+
+        await question.remove();
+        return true;
+    }
+
+    // Review Mutations
+    @Mutation(() => Review)
+    async submitReview(@Arg("input") input: SubmitReviewInput) {
+        const campaign = await ReviewCampaign.findOne({ where: { id: input.campaignId } });
+        if (!campaign) {
+            throw new Error("Review campaign not found");
+        }
+
+        const review = new Review();
+        review.reviewCampaign = campaign;
+        review.dateReviewCompleted = new Date();
+
+        await review.save();
+
+        // Create answers
+        for (const answerInput of input.answers) {
+            const answer = new ReviewAnswer();
+            answer.review = review;
+            answer.questionId = { id: answerInput.questionId } as ReviewQuestion;
+            answer.questionType = answerInput.questionType;
+            answer.textAnswer = answerInput.textAnswer || "";
+            answer.rangeAnswer = answerInput.rangeAnswer || 0;
+            answer.dropDownAnswer = answerInput.dropDownAnswerId ? { id: answerInput.dropDownAnswerId } as DropDown : null;
+
+            await answer.save();
+        }
+
+        return review;
+    }
+
+    @Mutation(() => Review)
+    async updateReview(@Arg("input") input: UpdateReviewInput) {
```

Comment:
review needs to be associated to either people that bought ticket_id, or people registration list